coffee_cmd/src/cmd.rs

```diff
@@ -29,6 +29,8 @@ pub enum CoffeeCommand {
         verbose: bool,
         #[arg(short, long, action = clap::ArgAction::SetTrue)]
         dynamic: bool,
+        #[arg(short, long, action = clap::ArgAction::SetTrue)]
```

Comment:
```suggestion
        #[arg(short, long)]
```
OK this argument is used randomly and we still did not understand for what, I look inside the docs https://docs.rs/clap/latest/clap/enum.ArgAction.html

and this in this case does not make sense